consensus/istanbul/validator/weighted.go

```diff
@@ -665,7 +665,7 @@ func (valSet *weightedCouncil) Refresh(hash common.Hash, blockNum uint64, config
 		return nil
 	}
 
-	totalStaking := calcTotalAmount(weightedValidators, newStakingInfo, stakingAmounts)
+	totalStaking, _ := calcTotalAmount(weightedValidators, newStakingInfo, stakingAmounts)
```

Comment:
This is just a minor question.. the returned `gini` is not used. Can I ask the reason where the returned `gini` is used?

Reply:
It is used for teting only. See https://github.com/klaytn/klaytn/pull/1464#discussion_r913382015.